crates/store/src/db/models/queries/accounts.rs

```diff
@@ -143,35 +176,46 @@ pub(crate) fn select_accounts_by_id(
     )
     .filter(schema::accounts::account_id.eq_any(account_ids))
     .load::<(AccountRaw, Option<Vec<u8>>)>(conn)?;
-    let account_infos = vec_raw_try_into::<AccountInfo, AccountWithCodeRaw>(
-        accounts_raw.into_iter().map(AccountWithCodeRaw::from),
+    let account_infos = vec_raw_try_into::<AccountInfo, AccountWithCodeRawJoined>(
+        accounts_raw.into_iter().map(AccountWithCodeRawJoined::from),
     )?;
     Ok(account_infos)
 }
 
+/// Select account vault assets within a block range.
```

Comment:
nit: matching the `select_accounts_by_block_range` docs
```suggestion
/// Select account vault assets within a block range (inclusive).
```